cmd/mungedocs/mungedocs.go

```diff
@@ -28,6 +29,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const latestReleaseBranch = "release-1.1"
```

Comment:
We need to update this when we made a new branch. This is the only maintenance cost.